modules/process/manager.go

```diff
@@ -6,13 +6,12 @@ package process
 
 import (
 	"bytes"
+	"context"
```

Comment:
Because some developers (@strk for example) still run 1.6. Latest version in Ubuntu 16.04 LTS is 1.6 as well http://packages.ubuntu.com/xenial/golang-go